docusaurus/docs/welcome/what-is-this.md

```diff
@@ -0,0 +1,19 @@
+---
+id: what-is-this
+title: What is this
+sidebar_label: What is this
+---
+
+This project is [MOXY's](https://moxy.studio/) boilerplate to accelerate the setup of new [**React Native**](https://reactnative.dev/) applications.
+
+## Motivation
+
+The goal of this project is to get your new React Native application started in seconds. It offers many features and it's built on industry best practices & standards.
+
+To start with our stack, we introduce you [**React Native**](https://reactnative.dev/).
+
+## React Native
+
+[React Native](https://reactnative.dev/) is an open-source mobile application framework. It is used to develop applications for Android, iOS, Web and UWP by enabling developers to use React along with native platform capabilities.
```

Comment:
```suggestion
React Native](https://reactnative.dev/) is an open-source mobile application development framework. It allows to develop applications for Android, iOS, Windows and other platforms by enabling developers to use React along with native platform capabilities.
```

Reply:
It is worth to mention that while React Native can actually be used to develop apps for Windows and other operating systems / platforms, our focus here is Android and iOS only (at least for the time being).

Reply:
Another thing worth mentioning is that our boilerplate was scaffolded with React Native's CLI which means we are not coupled to any framework built on top of React Native, e.g. Expo. While the setup is more complex in some aspects, it is also more flexible as we can easily make changes to native code and configurations.

Reply:
please renew your review since changes were made.

Reply:
Done! ✅